services/cronback-scheduler-srv/tests/scheduler_test.rs

```diff
@@ -27,6 +38,100 @@ use tonic::Request;
 use tracing::info;
 use tracing_test::traced_test;
 
+fn make_trigger(
+    name: impl Into<String>,
+    scheduled: bool,
+    description: impl Into<String>,
+) -> Trigger {
+    let schedule = if scheduled {
+        Some(Schedule {
+            schedule: Some(schedule::Schedule::Recurring(Recurring {
+                cron: "0 * * * * *".to_owned(),
+                timezone: "Europe/London".into(),
+                limit: Some(4),
+                ..Default::default()
+            })),
+        })
+    } else {
+        None
+    };
+
+    Trigger {
+        payload: Some(Payload {
+            body: "Hello World".into(),
+            ..Default::default()
+        }),
+        name: name.into(),
+        description: Some(description.into()),
+        action: Some(Action {
+            action: Some(action::Action::Webhook(Webhook {
+                url: "http://localhost:3000".to_owned(),
+                http_method: HttpMethod::Get.into(),
+                timeout_s: 30.0,
+                retry: None,
+            })),
+        }),
+        status: Default::default(),
+        schedule,
+        ..Default::default()
+    }
+}
+
+async fn install_trigger(
+    client_provider: &TestGrpcClientProvider<ScopedSchedulerClient>,
+    project: &ValidShardedId<ProjectId>,
+    trigger: Trigger,
+) -> UpsertTriggerResponse {
+    let upsert = UpsertTriggerRequest {
+        precondition: None,
+        trigger_name: None,
+        trigger: Some(trigger),
+    };
+    client_provider
+        .get_client(&RequestId::new(), project)
+        .await
+        .unwrap()
+        .upsert_trigger(Request::new(upsert))
+        .await
+        .unwrap()
+        .into_inner()
+}
+
+// make a list_triggers function similar to install_trigger
```

Comment:
super nit: Not sure of the value of this comment

Reply:
lol, self reminder. Will remove.